mix.exs

```diff
@@ -1,51 +1,85 @@
 defmodule TzWorld.Mixfile do
   use Mix.Project
 
+  @version "0.2.0"
+
   def project do
     [
       app: :tz_world,
       name: "TzWord",
-      version: "0.1.0",
-      elixir: "~> 1.3",
-      start_permanent: Mix.env == :prod,
+      version: @version,
+      elixir: "~> 1.8",
```

Comment:
Why did you update this?

Reply:
Yes that a bit aggressive of me to increase the elixir version - although I haven't tested on anything below 1.8. I've put it back to 1.3.

I extracted the `@version` because I use if further down to generate some package links which I've found helpful on hex in the past. Let me know if that's not ok.

Reply:
Ah, I recall now the reason I bumped it is because I'm using `handle_continue` in the `GenServer` which only came with OTP 21. And that requires [Elixir 1.6.6 or later](https://github.com/elixir-lang/elixir/blob/master/lib/elixir/pages/Compatibility%20and%20Deprecations.md). I've updated mix.exs to 1.6 for now.

Reply:
Also `defguard` comes with Elixir 1.6.